portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
```

Comment:
Please keep indentation consistent. Normally for HTML, I think we do 2-space indentation.